scripts/generate-query-library-rtk-apis.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * To generate query library k8s APIs, run:
+ * `npx rtk-query-codegen-openapi ./scripts/generate-query-library-rtk-apis.ts`
```

Comment:
Should this be somewhere near the QL API files? Or is this meant to be used later for all API generation?

Reply:
yeah i think that makes sense for now